doc/ch-image_desc.rst

```diff
@@ -231,6 +231,15 @@ Destination:
 
 Options:
 
+  :code:`--arch`
+    Specify architecture :code:`--arch=ARCH[/VARIANT]` for image described by
+    :code:`IMAGE_REF`. If found pull it; otherwise error. Use
+    :code:`--list-arch IMAGE_REF` for list of valid platform architectures.
```

Comment:
I think it would be good, if we can, to avoid separating the architectures into variants, and keep it at a single level, i.e., some architectures happen to include a slash.

```suggestion
  :code:`--arch`
    Specify architecture :code:`--arch=ARCH` for image described by
    :code:`IMAGE_REF`. If image exists but not as architecture
    :code:`ARCH`, fail with error.
```